content/pt-br/community/code-of-conduct.md

```diff
@@ -1,27 +1,30 @@
 ---
-title: Community
+title: Código de Conduta da Comunidade Kubernetes
 layout: basic
 cid: community
-css: /css/community.css
+community_styles_migrated: true
 ---
 
-<div class="community_main">
-<h1>Código de Conduta da Comunidade Kubernetes</h1>
-
+<div class="community-section" id="cncf-code-of-conduct-intro">
+<p>
 Kubernetes segue
-<a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">CNCF Código de Conduta</a>.
+<a href="https://github.com/cncf/foundation/blob/main/code-of-conduct.md">CNCF Código de Conduta</a>.
 O texto do CNC CoC é reproduzido abaixo, a partir de
-<a href="https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct.md">commit 214585e</a>.
+<a href="https://github.com/cncf/foundation/blob/fff715fb000ba4d7422684eca1d50d80676be254/code-of-conduct.md">commit fff715fb0</a>.
 Se você perceber que isso está desatualizado, por favor
 <a href="https://github.com/kubernetes/website/issues/new">abra um bug</a>.
```

Comment:
I prefer `issue` instead of `bug`, because the Github is `issue`. Another option would be "abra uma solicitação". Think about it.

Reply:
Hi @MrErlison, thank you for your review! 

Github docs translated "create an issue" as "criar um problema": https://docs.github.com/pt/issues/tracking-your-work-with-issues/creating-an-issue
Do you think it's a good idea to follow the docs translation?

Additionaly, I believe "crie uma issue" might also be a good solution, since it's commonly used.
Please let me know what you think.

Reply:
+1 "crie uma issue"

Reply:
Thanks! Updated in [8634438](https://github.com/kubernetes/website/pull/38407/commits/863443867eba418b25dc49bfbae30500236fd0eb)